nrfs/include/services/nrfs_usb.h

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef NRFS_USB_H
+#define NRFS_USB_H
+
+#include <internal/services/nrfs_usb.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** @brief USB service event types. */
+typedef enum __NRFS_PACKED {
+	NRFS_USB_EVT_VBUS_STATUS_CHANGE, /** VBUS voltage detection status has changed. */
+	NRFS_USB_EVT_REJECT,		 /** Request rejected. */
+} nrfs_usb_evt_type_t;
+
+/** @brief USB service event. */
+typedef struct {
+	nrfs_usb_evt_type_t type; /** Event type. */
+	bool vbus_detected;	  /** True if USB VBUS voltage is detected. */
+	bool vregusb_ok;	  /** True if VREG is running OK. */
+	bool usbhspll_ok;	  /** True if PLL is running OK. */
```

Comment:
Is it not the same as `nrfs_usb_rsp_data_t`?

Reply:
The `__NRFS_PACKED` structures are internal and backend related (sent through the backend). The ones containing the `nrfs_*****_evt_type_t type` are presented to the user, and are not packed.
 The 'packing/unpacking' is done by this very API lib.